<launch>
    <!--启动vicon-->
    <node pkg="vicon_bridge" type="vicon_bridge" name="vicon" output="screen">
		<param name="stream_mode" value="ClientPull" type="str" />
		<param name="datastream_hostport" value="192.168.1.2:801" type="str" />
		<param name="tf_ref_frame_id" value="world" type="str" />
	</node>
    <!--启动控制器-->
    <node pkg="square_run" type="run_square" name="turbot_ctrl_unit" output="screen">
        <param name="turtlebot_frame_id" value="vicon/turtlebot3/turtlebot3" type="str" />
        <param name="odom_frame_id" value="world" type="str" />
        <param name="control_topic" value="/cmd_vel" type="str" />

        <param name="pos_num" value="4" type="int"/>
        <param name="pos0_x" value="1.0" type="double"/>
        <param name="pos0_y" value="1.0" type="double"/>
        <param name="pos1_x" value="-1.0" type="double"/>
        <param name="pos1_y" value="1.0" type="double"/>
        <param name="pos2_x" value="-1.0" type="double"/>
        <param name="pos2_y" value="-1.0" type="double"/>
        <param name="pos3_x" value="1.0" type="double"/>
        <param name="pos3_y" value="-1.0" type="double"/>
    </node>
</launch>